airbyte-integrations/connectors/source-jira/sample_files/configured_catalog.json

```diff
@@ -0,0 +1,191 @@
+{
+  "streams": [
```

Comment:
unless there is stringent rate limiting, can we sync all streams? The more things we sync the more confident we are this works correctly. 

You may need to login to JIRA and create "fake" data like comments etc.. please let me know if you need support with any of that

Reply:
I added all the streams for testing during integration tests.
At the moment we are getting all the entities, I have created more than 50 fake comments for issues.

Reply:
fantastic. Thanks!